### ISSUE TITLE
mod_http2 not respecting mod_proxy with ProxyRequests On

### ISSUE THREAD

**Issue by bemasc** [icing/mod_h2#129]
I'm trying to set up a forward proxy with HTTP/2.  Unfortunately, it seems like mod_http2 just ignores mod_proxy if it has "ProxyRequests On".  Instead, http GET proxy requests just return index.html (!), and https proxy request (CONNECT) fail with some kind of protocol error.

Disabling mod_http2 makes everything work again, over HTTP/1.1.

Is there a trick to making this work?  Or is it just a bug?

**Comment by elukey** [comment]
Quick question - have you tried https://httpd.apache.org/docs/current/mod/mod_proxy_http2.html ?

**Comment by bemasc** [comment]
Yes, I have tried with mod_proxy_http2 enabled and disabled.  It doesn't seem to make any difference.

**Comment by icing** [comment]
@bemasc I think you have run into undiscovered country here. I have never tested ```mod_http2``` in a forward proxy setup. That it fails right at CONNECT is probably all to the good.

So, currently unsupported. You can make an "enhancement" issue out of this one, but I do not foresee to have myself time to work on this in the near future.

**Comment by icing** [comment]
This repository is being archived. Please report any issues to the [Apache httpd project](https://httpd.apache.org).
